docs/infrastructure_and_maintenance/databases.md

```diff
@@ -12,11 +12,6 @@ description: Ibexa DXP can use MySQL, PostgreSQL or MariaDB as its database.
 
 To use PostgreSQL, you need to have the `pdo_pgsql` PHP extension installed.
 
-Make sure you have the `ibexa/doctrine-dbal-schema` Composer package installed
```

Comment:
`ibexa/doctrine-dbal-schema` is not part of the default installation, not need for this warning.